cli/popper/commands/cmd_version.py

```diff
@@ -8,4 +8,5 @@
 @pass_context
 def cli(ctx):
     """Displays version of Popper and exit."""
+    pu.setup_logging()
```

Comment:
can we put this in [`cli.py`](https://github.com/systemslab/popper/blob/master/cli/popper/cli.py)? Then that would setup for the entire app